File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.py
answer = 0

for _ in range(int(input())):
    word, cnt = input(), 0

    for i in range(len(word) - 1):
        if word[i] != word[i + 1]:
            check = word[i + 1:]

            if check.count(word[i]):
                cnt += 1

    if not cnt:
        answer += 1

print(answer)

File: 백준/Silver/1316. 그룹 단어 체커/그룹 단어 체커.c
#include <stdio.h>
#include <string.h>

int main()
{
    int N;
    char word[100];
    scanf("%d", &N);
    int cnt = N;
    
    for (int i = 0; i < N; i++) {
        char first = '.';
        int arr[26] = {0};
        scanf("%s", word);
        
        for (int j = 0; word[j] != '\0'; j++) {
            if (first != word[j]) {
                first = word[j];
                arr[word[j] - 'a'] += 1;
            }
            
            if (arr[word[j] - 'a'] == 2) {
                cnt -= 1;
                break;
            }
        }
    }
    
    printf("%d", cnt);
}